### DISCUSSION TITLE
可以数据持久化吗

### DISCUSSION THREAD

**Discussion by mcratt** [SinKy-Yan/zhenxunbot-docker#15] [Q&A] [answered]
映射了3个文件夹，但是好像并不会释放文件，不做持久化docker寄了的话数据配置全寄
-v /volume1/docker/zhenxun_bot/go-cqhttp:/home/go-cqhttp \
-v /volume1/docker/zhenxun_bot/zhenxun_bot:/home/zhenxun_bot \
-v /volume1/docker/zhenxun_bot/zhenxun_bot_webui:/home/zhenxun_bot_webui \

**Comment by SinKy-Yan** [comment]
不会释放文件噢，映射了之后是把宿主机目录映射进来而不是把内部文件放出去

**Comment by SinKy-Yan** [comment, marked as answer]
新版已解决

**Reply by mcratt** [reply]
nb
